wpilib/wpilib/solenoidbase.py

```diff
@@ -22,14 +22,30 @@ def __init__(self, moduleNumber):
         :param moduleNumber: The PCM CAN ID
         """
         self.moduleNumber = moduleNumber
+        self.getAll = lambda: SolenoidBase.getAll(self.moduleNumber)
```

Comment:
I know it was discussed previously... but I've already forgotten, what's this weirdness for again? Since there are two functions called `getAll`...

Reply:
Also, wouldn't it be better to just do `self.getAll = SolenoidBase.getAll` (provided this is the feature you wanted

Reply:
the first declared getAll (line 32) is my feeble attempt to get the docs to show that you can call getAll from both static and nonstatic contexts with the different signatures. It probably won't work and if not should just be deleted

Reply:
Given I'm pretty sure Sphinx imports the code and introspects what's available at runtime, rather than doing static analysis, this almost certainly won't work.